plugins/chunter/src/__model__.ts

```diff
@@ -25,6 +25,7 @@ import presentation, { ComponentExtension } from '@anticrm/presentation'
 import { TVDoc, TEmb, TMixin } from '@anticrm/model/src/__model__'
 import { Ref, Class, Property } from '@anticrm/core'
 import { Application, VDoc } from '@anticrm/domains'
+import { WorkbenchApplication } from '@anticrm/workbench'
```

Comment:
I think since we have it imported for model, I think ideal will be to Fix `const chunter` definition with Ref<Application> to Ref<WorkbenchApplication> since no cast will be required in this case.